### ISSUE TITLE
Beef-Over-AWS Router Popup In Browser 

### ISSUE THREAD

**Issue by VortexVoltGIT** [AzizKpln/Beef-Over-AWS#6]
Hey
Whenever I open my public IP address up it pulled up my router configuration.
Since I use Verizon Router.
Is this an issue where the apache2 service is not starting up?
Here is the attached screenshot.
![image](https://user-images.githubusercontent.com/82333863/114315501-3ef2ba00-9acd-11eb-82ca-0aadf1691455.png)

**Comment by AzizKpln** [comment]
Hello,
Yes actually this is an issue because the malicous js code does not show up on this Page. When you activate a port for http connection on your router, the firewall of your router doesnt allow any request to be made to your Server.  This is a problem. You have to disable that security software and your server has to be able to request and response

**Comment by VortexVoltGIT** [comment]
So I do this in the router configuration??

**Comment by VortexVoltGIT** [comment]
And also can it be fixed by going into the port forwarding section of my router and inputting my public IP following port 3000 and allowing traffic for TCP&UDP?

**Comment by AzizKpln** [comment]
yes you have to do it in router configuration. For the second question, no port number 3000 is not related. Just port number 80 is required
